<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyBankcardWindow</class>
 <widget class="QDialog" name="MyBankcardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>311</width>
     <height>192</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Title</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Number</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_Delete">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>130</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>- - - - DELETE - - - -</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Title">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Title:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Number">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Title">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Number">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>add new Bankcard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>481</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string> close and save</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>220</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Title</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Number</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Date</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_OrderBy">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>221</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Order by:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Order">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>order</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
